README.md

```diff
@@ -1,25 +1,25 @@
-Every day, millions of people come up with ideas that have the potential to change the world. Yet almost all ideas do not take any form mainly because of lack of support from peers with similar vision or the originator does not have a clear path of how to make it a reality. 
+Every day millions of people come up with ideas that have the potential to change the world. Yet almost all ideas do not take any form, mainly because of lack of support from peers with similar vision, or the originator does not have a clear path of how to make it a reality. 
 
 > ## We exist to take your ideas to execution. 
 
-We are different from any freelancing program as in the process, you learn new skills and thousands of passionate people are beside your shoulder for the entire journey. You do not owe a penny to us. 
+We are different from any freelancing program as in the process you learn new skills and thousands of passionate people are beside your shoulder for the entire journey. You do not owe a penny to us. 
 
-> We do everything to bring a smile on your face. 😄 
+> We do everything to bring a smile to your face. 😄 
```

Comment:
This line was fine as it is.

Reply:
I'm not saying it's a big deal, it's just that the expression is 'bring a smile to [my/your] face'.

Reply:
@gavinmiller If you say so. The old version just sounded better to me.